src/sidebar/components/test/annotation-test.js

```diff
@@ -116,10 +131,11 @@ describe('annotation', function() {
     before(function() {
       angular.module('h', [])
         .component('annotation', annotationComponent())
+        .component('annotationActionButton', {
+          bindings: require('../annotation-action-button').bindings,
+        })
         .component('markdown', {
-          bindings: {
-            customTextClass: '<?',
-          },
+          bindings: require('../markdown').bindings,
```

Comment:
Correct. The reason for requiring the `bindings` from the _real_ annotation is so that the fake component will have the same interface as the real one - in terms of the type and names of its input properties.